From 25cace84c42c3cf0a754780d13378dd3b9942255 Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Fri, 22 Jun 2018 11:29:45 -0500
Subject: [PATCH 4108/5242] MLK-18669-1: arm64: dts: change the dummy cycle
 number

commit  4041ef942c26e49b10bb6dffdd0394368094f007 from
https://source.codeaurora.org/external/imx/linux-imx.git

change the dummy cycle number for quad ddr read

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
index f74a5fd..42654f7 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
@@ -492,7 +492,7 @@
 		#size-cells = <1>;
 		compatible = "micron,mt25qu256aba";
 		spi-max-frequency = <29000000>;
-		spi-nor,ddr-quad-read-dummy = <8>;
+		spi-nor,ddr-quad-read-dummy = <6>;
 	};
 };
 
-- 
1.7.9.5

